clang/include/clang/Format/Format.h

```diff
@@ -2218,6 +2218,11 @@ struct FormatStyle {
   /// \version 3.8
   bool BreakAfterJavaFieldAnnotations;
 
+  /// If ``true``, clang-format will always break before function definition
+  /// parameters
+  /// \version 19
+  bool BreakFunctionDefinitionParameters;
```

Comment:
Please sort your code on all locations directly after `BreakConstructorInitializers`. It seems there is something unsorted.

Reply:
Not only here, but also in the `operator==`, `mapping` and `getLLVMStyle` function.